File: android/app/src/main/java/com/texasgamer/zephyr/model/WhatsNewItem.java
package com.texasgamer.zephyr.model;

import androidx.annotation.DrawableRes;
import androidx.annotation.StringRes;

/**
 * Represents item in "What's new" list.
 */
public class WhatsNewItem {

    @DrawableRes
    private int mIconRes;
    @StringRes
    private int mTitleRes;
    @StringRes
    private int mBodyRes;

    public WhatsNewItem(@DrawableRes int iconRes, @StringRes int titleRes, @StringRes int bodyRes) {
        mIconRes = iconRes;
        mTitleRes = titleRes;
        mBodyRes = bodyRes;
    }

    @DrawableRes
    public int getIconRes() {
        return mIconRes;
    }

    @StringRes
    public int getTitleRes() {
        return mTitleRes;
    }

    @StringRes
    public int getBodyRes() {
        return mBodyRes;
    }
}

File: android/app/src/main/java/com/texasgamer/zephyr/model/WhatsNewItem.kt
package com.texasgamer.zephyr.model

import androidx.annotation.DrawableRes
import androidx.annotation.StringRes

/**
 * Represents item in "What's new" list.
 */
class WhatsNewItem(@field:DrawableRes @get:DrawableRes
                   @param:DrawableRes val iconRes: Int, @field:StringRes @get:StringRes
                   @param:StringRes val titleRes: Int, @field:StringRes @get:StringRes
                   @param:StringRes val bodyRes: Int)